### DISCUSSION TITLE
Can one do multicam with BRAW Toolbox?

### DISCUSSION THREAD

**Discussion by Doctor-Chas** [latenitefilms/BRAWToolbox#166] [Support] [answered]
Given the delightfully quirky method Toolbox uses to create a workable file within FCP, is it possible to do actual Multicam with it?

In other words, can one take two or more BRAW clips shot from different angles and make another Multicam clip that can be edited in the usual manner?

**Comment by latenitefilms** [comment, marked as answer]
Yes - see: https://brawtoolbox.io/troubleshooting/#how-can-i-work-with-multicam-clips
